### ISSUE TITLE
Mixed-Exponential Jump Diffusion

### ISSUE THREAD

**Issue by stevenlvkai** [jirotubuyaki/Jdmbs#1]
Why you use a uniform random variable multiplying a normal random variable as Mixed-Exponential Jump Diffusion?
`J[j] <- J[j] + runif(1,min=0,max=1) * rnorm(1, mean = 0, sd = 1)`

**Comment by jirotubuyaki** [comment]
Thank you for pointing out the issue.
I implemented it for random jump size. But even if I remove "runif(1,min=0,max=1)" , Random jumps will occur.
Do you think this codes have problems?

**Comment by stevenlvkai** [comment]
> Thank you for pointing out the issue.
> I implemented it for random jump size. But even if I remove "runif(1,min=0,max=1)" , Random jumps will occur.
> Do you think this codes have problems?

No, I don't think this is a problem. I just don't understand why using a uniform distribution here. It seems like the uniform distribution just shrinks the amplitude of the jump. As you said, rnorm() has already provided a random generator. So, why we need a uniform variable to adjust it? Is there any paper mentions this method?

**Comment by jirotubuyaki** [comment]
I mainly refer "Stochastic Calculus for Finance II: Continuous-Time Models (Springer Finance)".
But this book is not mention jump size and I don't refer any academic papers.

**Comment by stevenlvkai** [comment]
> I mainly refer "Stochastic Calculus for Finance II: Continuous-Time Models (Springer Finance)".
> But this book is not mention jump size and I don't refer any academic papers.

Ok, understand. Thank you very much. I really appreciate your work. Maybe you can mention a little more detail about $_Yi_$ in the mixed-exponential jump-diffusion model in your vignette.

**Comment by jirotubuyaki** [comment]
I will better my repository and vignette.
Thank you very much.
